client/cl_bennys.lua

```diff
@@ -546,6 +546,10 @@ function ApplyMod(categoryID, modID)
         originalMod = modID
 
         SetVehicleMod(plyVeh, categoryID, modID)
+		
+	if categoryID == 48 then
```

Comment:
```suggestion
        if categoryID == 48 then
```
Fix the indentation here